management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatistics.java

```diff
@@ -221,17 +220,21 @@ public void dispose() {
 
 
   @SuppressWarnings({"unchecked", "rawtypes"})
-  private ConcurrentMap<String, OperationStatistic<?>> discoverCountStatistics() {
-    ConcurrentHashMap<String, OperationStatistic<?>> result = new ConcurrentHashMap<String, OperationStatistic<?>>();
```

Comment:
Why would you use a ConcurrentMap here, that, besides, depend on the internal impl of Ehcache ? It's just a local map which is created sequentially, then read-only.